packages/unlock-js/openapi.yml

```diff
@@ -381,7 +381,7 @@ paths:
               type: object
               nullable: true
               properties:
-                refresh-token:
```

Comment:
Is this another PR?

Reply:
No, just saw a small spelling fix so added it.